package-lock.json

```diff
@@ -0,0 +1,8046 @@
+{
+  "name": "@salesforce/apex-node",
```

Comment:
I got some errors around this during build that yarn was confused about both a package-lock.json and a yarn.lock, so I think we may want to remove this in favor of yarn.lock

Reply:
Done! That build error went away after removing.